install_config/install/deploy_router.adoc

```diff
@@ -1030,6 +1030,57 @@ $ echo "    router metrics at: http://<ip>:9090/consoles/haproxy.html "
 ====
 
 
+=== Preventing Connection Failures During Restarts
+
+If you connect to the router while the proxy is reloading, there is a
+small chance that your connection will end up in the wrong network
+queue and be dropped.  The issue is being addressed.  In the meantime,
+it is possible to work around the problem by installing *_iptables_*
```

Comment:
I feel like we have mixed usage of markup for iptables, in general. So maybe size up and choose the best one.